### PR TITLE
Update licensing-data.md with content created in another branch

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Addresses a sub-task in #1075 created in #1194

In this PR, I make a start with the chapter on Data Licenses. 

### List of changes proposed in this PR (pull-request)

* Brief intro to data licenses
* Table with permission levels
* CC0 license

Edit:
More changes in subsequent commits:
* Corrections to section on CC licences
* ODC licenses
* Comparing CC and ODC licenses
* Other licensing options specific to communities or use cases

---October 2020---
* Addressed concerns raised by PR reviewers
* Converting HTML table to Markdown
* Adding references to the references.bib file

### What should a reviewer concentrate their feedback on?

* Clarity of content
* Adequate to make into a chapter?
* Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [X] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: agarwalmanvi

### CODE REVIEW

**Comment by malvikasharan** [comment]
Suggestion to consider from #1194 

As commented by me
> I see that you have covered some important ground. However, I will wait to merge it as there are a few areas of Data Licensing that needs to be covered in this chapter. We have a reference from our colleagues from DCC (see here: https://www.dcc.ac.uk/guidance/how-guides/license-research-data).

As commented by @rosiehigman 
>Line 11: Creative Commons licences are not specifically data licences, they are just widely used as they are easy to understand, it would be helpful to clarify this.
> Line 48: CC0 is a dedication rather than a licence. It might be worth adding explicitly that you forfeit the legal right to attribution through CC0 and instead rely on academic norms.
> A general suggestion would be to think about mentioning/linking to the Open Data Commons licences (https://opendatacommons.org/licenses/index.html) and the Open Government Licence (http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/). The latter is quite UK-specific but is applied to a lot of data from public bodies and so is useful for researchers to be aware of if they are re-using data.
It would be a good idea to go through and check for licence (which is the noun so should be used for CC licences etc) and license (which is a verb and so used for the act of licensing data).

As commented by @c-martinez
> I agree with @rosiehigman point and that changing it en masse might be best. I would even say that we could focus in this PR on data licensing, and do a separate PR dealing with the licence vs license topic.

File: book/website/reproducible-research/licensing/licensing-data.md [hunk 1]
```diff
@@ -1,3 +1,103 @@
...
 # Data licenses
-This section is under construction.
+
+Similar to a software license, a data license governs what someone else can do with data that you create or own and that you make accessible to others through, for example, a data repository.
+Data licenses vary based on different criteria, such as:
+* Attribution to original owner
+* Permission to redistribute or modify original
+* Inclusion of the same license with derivatives or redistributions
+
+As a result, accessibility to your data is affected by the data license you choose. <br>
```
Comment (malvikasharan) [inline book/website/reproducible-research/licensing/licensing-data.md:10]:
```suggestion
As a result, accessibility to your data is affected by the data license you choose.
```

File: book/website/reproducible-research/licensing/licensing-data.md [hunk 2]
```diff
@@ -1,3 +1,103 @@
...
+        </tr>
+        <tr>
+            <th>NC</th>
+            <td>Only non-commerical uses are allowed</td>
+        </tr>
+        <tr>
+            <th>ND</th>
+            <td>No derivatives are allowed</td>
+        </tr>
+    </tbody>
```
Comment (malvikasharan) [inline book/website/reproducible-research/licensing/licensing-data.md:47]: change this to markdown formatting.

**Review by ahornos — APPROVED** [review]

**Comment by malvikasharan** [comment]
@all-contributors please add @ahornos for review

**Comment by malvikasharan** [comment]
@paulowoicho please go ahead and merge it when you are ready.

**Pull request merged**